### PR TITLE
Add jpkrohling as owner for some components

### PR DESCRIPTION
**Pull request by jpkrohling**
- 1st commit adds jpkrohling to a few components
- 2nd commit sorts entries alphabetically

Related to #3870

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -40,19 +44,18 @@ exporter/signalfxexporter/                           @open-telemetry/collector-c
...
 exporter/stackdriverexporter/                        @open-telemetry/collector-contrib-approvers @aabmass @dashpole @jsuereth @punya @tbarker25
-exporter/googlecloudexporter/                        @open-telemetry/collector-contrib-approvers @aabmass @dashpole @jsuereth @punya @tbarker25
 exporter/sumologicexporter/                          @open-telemetry/collector-contrib-approvers @pmm-sumo @sumo-drosiek
-exporter/uptraceexporter/                            @open-telemetry/collector-contrib-approvers @vmihailenco
-exporter/googlecloudpubsubexporter/                  @open-telemetry/collector-contrib-approvers @alexvanboxel
 exporter/tanzuobservabilityexporter/                 @open-telemetry/collector-contrib-approvers @oppegard @thepeterstone @keep94
 exporter/tencentcloudlogserviceexporter/             @open-telemetry/collector-contrib-approvers @wgliang @yiyang5055
-exporter/influxdbexporter/                           @open-telemetry/collector-contrib-approvers @jacobmarble @8none1
+exporter/uptraceexporter/                            @open-telemetry/collector-contrib-approvers @vmihailenco
```
Comment (mx-psi) [inline .github/CODEOWNERS:50]: Looks like this was removed in #4458 so we can delete this line

**Review by mx-psi — APPROVED** [review]
Approving, but left some comments that can either be dealt here or in a separate PR

**Comment by jpkrohling** [comment]
Given the potential for conflicts, I'll merge this as is and send in a new PR with the requested changes.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -83,14 +87,19 @@ receiver/cloudfoundryreceiver/                       @open-telemetry/collector-c
 receiver/collectdreceiver/                           @open-telemetry/collector-contrib-approvers @owais
 receiver/dockerstatsreceiver/                        @open-telemetry/collector-contrib-approvers @rmfitzpatrick
 receiver/dotnetdiagnosticsreceiver/                  @open-telemetry/collector-contrib-approvers @pmcollins @davmason
+receiver/filelogreceiver/                            @open-telemetry/collector-contrib-approvers @djaglowski
 receiver/fluentforwardreceiver/                      @open-telemetry/collector-contrib-approvers @dmitryax
+receiver/googlecloudpubsubexporter/                  @open-telemetry/collector-contrib-approvers @alexvanboxel
```
[the remark this answers is not in the document]
Comment (jpkrohling) [reply .github/CODEOWNERS:92]: Created https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6539

**Pull request merged**